src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,41 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_PASVORD_LENDG = 6;
+    private static final int MINIMUM_LOGIN_LENDG = 6;
+    private static final int MINIMUM_AEG = 18;
```

Comment:
There's a typo in the constant name `MINIMUM_AEG`. It should be `MINIMUM_AGE` to follow the naming convention and correct spelling.